pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -287,6 +287,27 @@ func (b *HybridBotanist) DeployKubeAPIServer() error {
 				admissionPlugins = append(admissionPlugins, plugin)
 			}
 		}
+
+		if apiServerConfig.AuditConfig != nil && apiServerConfig.AuditConfig.AuditPolicy != nil {
+
+			cmRefName := apiServerConfig.AuditConfig.AuditPolicy.Name
+			cmRefNamespace := b.Shoot.Info.Namespace
+			auditPolicyCm, err := b.K8sGardenClient.GetConfigMap(cmRefNamespace, cmRefName)
+			if err != nil {
+				return err
+			}
+			auditPolicy, ok := auditPolicyCm.Data["policy"]
+			if !ok {
+				return fmt.Errorf("Missing '.data.policy' in audit policy configmap %v/%v", cmRefNamespace, cmRefName)
+			}
+			if len(auditPolicy) == 0 {
+				return fmt.Errorf("Invalid audit policy. Provide non-empty audit policy")
+			}
+			// TODO: check provided policy is valid audit policy according to "k8s.io/apiserver/pkg/apis/audit[v1alpha1|v1beta1|v1]"
```

Comment:
Cool, nice idea! What about this TODO? Can't we just call https://github.com/kubernetes/apiserver/blob/master/pkg/apis/audit/validation/validation.go#L28?